packages/Button/src/button.scss

```diff
@@ -497,7 +501,12 @@
       }
 
       .ids-btn--mobile {
-        width: auto;
+        padding: 0;
+        width: var(--ids-btn-width);
```

Comment:
You could have made it with `aspect-ratio: 1/1;` and not having to declare 4 variables.

Reply:
hmm you have a point. I'll try this, thanks